committee-steering/governance/sig-governance-template-short.md

```diff
@@ -50,6 +50,13 @@ Membership for roles tracked in: <link to OWNERS file>
   - *MAY* participate in decision making for the subprojects they hold roles in
   - Includes all reviewers and approvers in [OWNERS] files for subprojects
 
+- Security Contact
+  - *MUST* be a contact point for the Product Security Team to reach out to for
+    triaging and handling of incoming issues
+  - *MUST* accept the [Embargo Policy](https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#embargo-policy)
+  - Defined in `SECURITY_CONTACTS` files, this is only relevant to the root file in
+    the repository
```

Comment:
Do you want to define the syntax here (one GitHub username per line, with blank lines and `#` comments ignored)?  Or just link to the template once kubernetes/kubernetes-template-project#18 lands and assume the sytax is self-evident?  Without at least one of those, examples in the wild may not be consistent enough for machine parsing.